package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "sfmovies",
+  "version": "0.0.0",
+  "description": "A RESTful API for movies filmed in San Francisco",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "lint": "eslint .",
+    "release:patch": "changelog -p && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
+    "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
+    "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
+    "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
+    "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000",
+    "start": "nodemon --ignore test/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sjl2/sfmovies.git"
+  },
+  "author": "Stewart Lynch",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/sjl2/sfmovies/issues"
+  },
+  "homepage": "https://github.com/sjl2/sfmovies#readme",
+  "devDependencies": {
+    "chai": "^3.0.0",
+    "eslint": "^1.9.0",
+    "eslint-config-lob": "^2.0.0",
+    "generate-changelog": "^1.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.2.5",
+    "nodemon": "^1.9.2"
+  },
+  "dependencies": {
+    "hapi": "^13.5.0"
+  }
```

Comment:
nit: It's the generators fault (https://github.com/lob/generator/issues/11) but could you move the `dependencies` above `devDependencies` and order the npm `scripts` alphabetically to keep the `package.json` consistent with our other repositories?